spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'case sensitive email' do
+    let!(:user) { User.create(email: 'test@test.com', password: '123123123') }
+
+    it 'does not create a user with the same email' do
+      new_user = User.new(email: 'test@test.com', password: '123123123')
+      expect(new_user.valid?).to be_falsey
+    end
+
+    it 'does not create a user with the same email different case' do
+      new_user = User.new(email: 'Test@test.com', password: '123123123')
+      expect(new_user.valid?).to be_falsey
+    end
```

Comment:
Can you edit a user? Does this validation workis in edit? more test for user management